ferdium/ferdium-app
# Switch npm to pnpm

#### Pre-flight Checklist

1. Please remember that if you are logging a bug for some service that has _stopped working_ or is _working incorrectly_, please log the bug [here](https://github.com/ferdium/ferdium-recipes/issues)
2. If you are requesting support for a **new service** in Ferdium, please log it [here](https://github.com/ferdium/ferdium-recipes/pulls)
3. Please remember to read the [self-help documentation](https://github.com/ferdium/ferdium-app#troubleshooting-recipes-self-help) - in case it helps you unblock yourself for issues related to older versions of recipes that were installed on your machine. (These will get automatically upgraded when you upgrade to the newer versions of Ferdium, but to get new recipes between Ferdium releases, this documentation is quite useful.)
4. Please ensure you've completed all of the following.

- [x] I have read the [Contributing Guidelines](https://github.com/ferdium/ferdium-app/blob/develop/CONTRIBUTING.md) for this project.
- [x] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-app/blob/develop/CODE_OF_CONDUCT.md) that this project adheres to.
- [x] I have searched the [issue tracker](https://github.com/ferdium/ferdium-app/issues) for a feature request that matches the one I want to file, without success.

#### Description of Change

Move electron-builder outside of package.json script. As using inside is causing trouble in native buildings. The same model we already followed to fix windows building with pnpm

#### Checklist


- [x] My pull request is properly named
- [x] The changes respect the code style of the project (`pnpm prepare-code`)
- [x] I tested/previewed my changes locally

#### Testing
I have done testing on macOS

**Steps to test**
- Remove `--$TARGET_ARCH --$TARGET_OS --dir` from the build scripts
- Run CLEAN=true ./scripts/build-unix.sh
- Once done, copy arm Ferdi.app to M1 machine and verify if no error thrown in console and vice versa (Build in arm and copy amd Ferdi.app to intel and verify no errors thrown)
